resources/js/Pages/Admin/Rooms/UpdateRoomForm.vue

```diff
@@ -1,98 +1,195 @@
+                    <input type="checkbox" id="checkbox" class="form-checkbox" v-model="form.alcohol"/>
+                    <jet-input-error :message="form.error('alcohol')" class="mt-2"/>
+                </div>
 
+                <div class="m-6">  
+                    <jet-label for="a_v_permitted" value="A/V Equipment Permitted"/>
+                    <input type="checkbox" id="checkbox" class="form-checkbox" v-model="form.a_v_permitted"/>
+                    <jet-input-error :message="form.error('a_v_permitted')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="status" value="Status"/>
+                    <select v-model="form.status" class="mt-1 block w-full" name="status" id="status">
+                        <option value="" selected disabled hidden>Select Room Status</option>
+                        <option value="available">Available</option>
+                        <option value="unavailable">Unavailable</option>
+                    </select>
+                    <jet-input-error :message="form.error('status')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="equippment" value="Equippment"/>
+                    <br>
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.projector"/>
+                    <span class="ml-2 text-sm text-black">Projector</span>    
+                    </label> 
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.television"/>
+                    <span class="ml-2 text-sm text-black">Television</span>  
+                    </label> 
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.computer"/>
+                    <span class="ml-2 text-sm text-black">Computer</span>
+                    </label>   
+                    <label class="flex items-center"> 
+                    <input type="checkbox" class="form-checkbox" v-model="form.whiteboard"/>
+                    <span class="ml-2 text-sm text-black">Whiteboard</span>    
+                    </label>                 
+                    <jet-input-error :message="form.error('equippment')" class="mt-2"/>
+                </div>
+
+                <div class="m-6">  
+                    <jet-label for="restrictions" value="Restrictions"/>
+                    <br>
+                    <label class="flex items-center">
+                    <input type="checkbox" class="form-checkbox" v-model="form.ambiant_music"/>
+                    <span class="ml-2 text-sm text-black">Ambiance Music</span>    
+                    </label> 
+                    <input type="checkbox" class="form-checkbox" v-model="form.sale_for_profit"/>
```

Comment:
```suggestion
                    <label class="flex items-center">
                    <input type="checkbox" class="form-checkbox" v-model="form.sale_for_profit"/>
```
Missing label tag.
![image](https://user-images.githubusercontent.com/21128579/105226387-ec3cde00-5b2d-11eb-8605-9cca659cf0f9.png)

Reply:
Perfect.